inventory.schema.json

```
@@ -2051,6 +2051,10 @@
                   "redash/redash"
                 ]
               },
+              "ipaddress": {
+                "type": "string",
+                "title": "Virtualmachine ip"
+              },
```

```suggestion
              "ipaddress": {
                "type": "string",
                "format": "ipv4",
                "title": "Virtualmachine ip",
                "examples": [
                  "127.0.0.1"
                ],
              },
```
In addition :
do you know if it's possible to recover IPV6 or not?

---

I think we should not limit the format to any version of IP. We already have several existing cases: with two fields, or with a `anyOf` condition (I guess this one is preferable).

---

Made some tests with `anyOf` and it seems to work as expected.